tekton/cronjobs/bases/image-build/trigger-image-build.yaml

```diff
@@ -101,7 +101,7 @@ spec:
               - name: SINK_URL
                 value: "sink-url"
               - name: TARGET_IMAGE
-                value: "gcr.io/tekton-releases/dogfooding/myimage"
+                value: "ghcr.io/tekton-releases/dogfooding/myimage"
```

Comment:
Should these references be updated to `ghcr.io/tektoncd/plumbing/…` to avoid confusion?

There are a bunch of other references to `tekton-releases` which are no longer valid as far as I understand.

Similar for references to `tekton-nightly` which I think is no longer used, as the nightly releases are no longer namespaced separately.